power_grid_model_c/power_grid_model/include/power_grid_model/component/branch.hpp

```diff
@@ -144,13 +144,21 @@ class Branch : public Base {
     }
 
     template <symmetry_tag sym> BranchOutput<sym> get_null_output() const {
-        BranchOutput<sym> output{};
+        BranchOutput<sym> output{.loading = {},
```

Comment:
i think that some compilers will complain that there are missing initializers on this.

Reply:
No. If the struct has all default values defined, compiler will not complain.